features/mesh/policies/Index.feature

```diff
@@ -147,3 +147,12 @@ Feature: mesh / policies / index
     And the "[data-testid='policy-type-link-MeshFaultInjection']" element exists
     # Always shows MeshGateway
     And the "[data-testid='policy-type-link-MeshGateway']" element exists
+
+  Scenario: Regression test: Zone column is visible when navigating from legacy policy type
```

Comment:
Aren't all tests regression tests? i.e. bit of a nit, but do we need this prefix?

Reply:
I think it’s useful to know when a test is dedicated to a regression.